app/resources/forms.js

```diff
@@ -158,6 +158,8 @@ function translate(source, target, text, continuation) {
       } else if (target.length > 2) {
         // Try falling back to "fr" if "fr-CA" didn't work.
         translate(source, target.slice(0, 2), text, continuation);
+      } else {
+        console_error("Call to Translate API has failed: ", response);
```

Comment:
Are you no longer checking translate_api_key?

Is it based on the assumption that translate_api_key should always be set?